src/js/components/settings/settings.js

```diff
@@ -11,13 +11,53 @@ import { Elements } from '@stripe/react-stripe-js';
 import { getCurrentUser, getIsEnterprise, getUserRoles } from '../../selectors';
 import SelfUserManagement from '../user-management/selfusermanagement';
 import UserManagement from '../user-management/usermanagement';
+import Global from './global';
+import Integrations from './integrations';
 import Organization from './organization';
 import Roles from './roles';
-import Global from './global';
 import Upgrade from './upgrade';
 
 let stripePromise = null;
 
+const sectionMap = {
+  'global-settings': { admin: false, enterprise: false, multitenancy: false, userManagement: false, component: <Global />, text: () => 'Global settings' },
+  'my-profile': { admin: false, enterprise: false, multitenancy: false, userManagement: false, component: <SelfUserManagement />, text: () => 'My profile' },
+  'organization-and-billing': {
+    admin: false,
+    enterprise: false,
+    multitenancy: true,
+    userManagement: false,
+    component: <Organization />,
+    text: () => 'Organization and billing'
+  },
+  'user-management': {
+    admin: false,
+    enterprise: false,
+    multitenancy: false,
+    userManagement: true,
+    component: <UserManagement />,
+    text: () => 'User management'
+  },
+  'role-management': { admin: true, enterprise: true, multitenancy: false, userManagement: false, component: <Roles />, text: () => 'Roles' },
+  'integrations': {
+    admin: true,
+    enterprise: false,
+    multitenancy: false,
+    userManagement: false,
+    component: <Integrations />,
+    text: () => 'Integrations'
+  },
```

Comment:
Don't we need to add a menu item "Integrations" in the user dropdown, too?

![image](https://user-images.githubusercontent.com/1295287/142354151-adebdb68-5159-4078-a5ed-21e16e011357.png)